system/lib/sbrk.c

```diff
@@ -11,7 +11,9 @@
 #endif
 #include <limits.h>
 #include <stddef.h>
+#include <stdlib.h>
 #include <stdint.h>
+#include <stdio.h>
```

Comment:
These newly added headers should probably also be behind an `#if __EMSCRIPTEN_PTHREADS__`.